Publish cloud.gov Pages CRM (#2207)
18F/cg-site: _posts/2022-11-15-pages-crm-release.md

---
layout: post
title: Release of cloud.gov Pages Customer Responsibility Matrix (CRM)
date: November 15, 2022
excerpt: As part of the launch of cloud.gov Pages we are 
  publishing the Pages-specific Customer Responsibility Matrix (CRM)
---

Our website publishing platform, [cloud.gov

> Pages](https://cloud.gov/pages/), is now fully authorized by

Reviewer: ```suggestion
Pages]({{ site.baseurl }}pages/), is now fully authorized by
```

[FedRAMP®](https://fedramp.gov) at the Moderate impact level, and
is replacing the Federalist offering, formerly at
https://federalist.18f.gov. As part of that migration, we are

> publishing the **[Control Implementation Summary (CIS) + Customer Responsibility Matrix (CRM) + Control-by-Control Inheritance (.xlsx)]({{ site.baseurl }}resources/cloud.gov-Pages-CIS-Worksheet.xlsx)**.

Reviewer: ```suggestion
publishing the **[Control Implementation Summary (CIS) + Customer Responsibility Matrix (CRM) + Control-by-Control Inheritance (.xlsx)]({{ site.baseurl }}/resources/cloud.gov-Pages-CIS-Worksheet.xlsx)**.
```

The CRM is a summary of each Low security control and whether it
is handled by cloud.gov, a shared responsibility, or a customer
responsibility. It includes guidance on which controls a customer
system can fully or partially inherit from cloud.gov Pages.

We will release a CRM for the cloud.gov Pages Moderate impact controls in the near future. Updates to the cloud.gov Pages CRM will

> be reflected on our [FedRAMP Tracker page]({{ site.baseurl }}/overview/fedramp-tracker/)

Reviewer: ```suggestion
be reflected on our [FedRAMP Tracker page]({{ site.baseurl }}/docs/overview/fedramp-tracker/)
```
